### ISSUE TITLE
CAN FD Hardware Support for VECTOR XL

### ISSUE THREAD

**Issue by RoSchCo** [rbei-etas/busmaster#1044]
Is there a planning for CAN FD and VECTOR XL or will only ETAS Hardware be the Interface for FD?
Thx in advance.

**Comment by aptj-sakurai** [comment]
I am also interested in this. Could you please advise if there's any schedule for VECTOR XL and/or PCAN-USB FD (see https://github.com/rbei-etas/busmaster/issues/885)?
Thank you!

**Comment by ETAS-Nithin** [comment]
Yes, CAN FD Hardware Support for VECTOR devices shall be available as part of the CAN FD commercial add-on http://rbei-etas.github.io/busmaster/Add_Ons.html
